plenum/server/node.py

```diff
@@ -550,10 +558,24 @@ def onStopping(self):
         self.nodestack.stop()
         self.clientstack.stop()
 
+        self.closeAllLevelDBs()
+
         self.mode = None
         if isinstance(self.poolManager, TxnPoolManager):
-            self.ledgerManager.setLedgerState(0, LedgerState.not_synced)
-        self.ledgerManager.setLedgerState(1, LedgerState.not_synced)
+            self.ledgerManager.setLedgerState(POOL_LEDGER_ID,
+                                              LedgerState.not_synced)
+        self.ledgerManager.setLedgerState(DOMAIN_LEDGER_ID,
+                                          LedgerState.not_synced)
+
+    def closeAllLevelDBs(self):
```

Comment:
Rename the method; get rid of using LevelDB, since it looks like a generic close method